imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -54,6 +54,25 @@ function getSearchLanguage() {
   return { default_language: "en" };
 }
 
+/**
+ * handleRawCollectionFailures
+ * When using Collection.rawCollection() methods that return a Promise,
+ * handle the errors in a catch. However, ignore errors with altering indexes
+ * before a collection exists.
+ * @param  {Error} error an error object returned from a Promise rejection
+ * @return {undefined}   doesn't return anything
+ */
+function handleRawCollectionFailures(error) {
+  // If we get an error from the Mongo driver because something tried to drop al
```

Comment:
typo in comment

Reply:
Yeah, I fixed that 3 days ago.  https://github.com/reactioncommerce/reaction/pull/3243/commits/881f2dc3243d1143a637f789f8178da7b46e5420

Not sure how that was still showing here.